policyengine_us/variables/gov/hhs/tanf/cash/eligibility/is_person_demographic_tanf_eligible.py

```diff
@@ -9,7 +9,10 @@ class is_person_demographic_tanf_eligible(Variable):
 
     def formula(person, period, parameters):
         child_0_17 = person("is_child", period)
-        is_18 = person("age", period) == 18
+        is_18 = (
```

Comment:
Instead add `19` as a parameter for student eligibility and use that here.

Reply:
would this work instead or would it be better to include additional parameter of 19?

Reply:
add a parameter please, since the legal code has it as a separate number